packages/miew/package.json

```diff
@@ -19,7 +19,7 @@
   "license": "MIT",
   "scripts": {
     "ci": "run-s clean lint test-cover build",
-    "ci-fast": "run-s lint:js test:unit build:demo",
+    "ci-fast": "run-s lint:js test:unit build",
```

Comment:
The intention of "fast" was to build only minimal configuration for development. Do we really need to change it? Perhaps, remove it completely if we don't use it anymore?